gtsfm/scene_optimizer.py

```diff
@@ -94,7 +95,7 @@ def __init__(
         # make directories for persisting data
         os.makedirs(PLOT_BASE_PATH, exist_ok=True)
         os.makedirs(METRICS_PATH, exist_ok=True)
-        os.makedirs(RESULTS_PATH, exist_ok=True)
+        os.makedirs(RESULTS_PATH , exist_ok=True)
```

Comment:
small edit accidentally made here. running `black gtsfm/scene_optimizer.py` will undo it.

Reply:
Fixed

Reply:
Done.